README.md

```diff
@@ -259,8 +266,3 @@ $ sudo tee -a /usr/share/applications/gns3.desktop > /dev/null << EOL
 > Categories=Application;Network;Qt;
 > EOL
 ```
-
-# Changelog
-
-June 29th 2019
-* As usual, there were a ton of minor bugs in the initial commit. The script should now be completely functional. Some cosmetic changes will be added to make the script more readable.
```

Comment:
I'd prefer to also keep change log in the README. That said it can be updated to reflect changes contributed by you  - which I could do as well :)